### DISCUSSION TITLE
Jakiper battery

### DISCUSSION THREAD

**Discussion by jameshowison** [Louisvdw/dbus-serialbattery#562] [New BMS requests]
Would be great to have support for Jakiper server rack batteries.

They have both the RS232 and RS485 ports.

https://jakiperbattery.com/2021/09/22/jk48v100-rs232-pc-monitoring-software/

The BMS is possibly using the PACE protocol?  Just judging by this thread, I have no inside info :)

https://diysolarforum.com/threads/jakiper-48v-sok-battery-bms-is-pace-brand-and-not-seplos-type.38015/

**Comment by larduino** [comment]
Also look at this info
 https://diysolarforum.com/threads/monitoring-jakiper-maybe-sok-too-server-rack-batteries-via-modbus-to-esp8266-mqtt.45540/

**Comment by larduino** [comment]
A monitoring program for Jakiper / SOK batteries has been made.  See this project which also has wiki on how to install using either Raspberry Pi  or using an Esp32  https://github.com/ClassicDIY/PylonToMQTT
